src/modules/listOfTasks.js

```diff
@@ -1,19 +1,25 @@
-import tasks from './tasks.js';
+// import tasks from './tasks.js';
```

Comment:
- [ ] **Optional:** Kindly remove all commented and unused code to have a more professional codebase.